source/ja/api_classifier.rst

```diff
@@ -101,6 +103,29 @@ JSON の各フィールドは以下のとおりである。
 
         * 値域: 0.0 < ``regularization_weight``
 
+    NN
```

Comment:
Ah, this must be 3 spaces.
Indentation goes wrong with 4 spaces.
http://ci.jubat.us/job/website/ws/Branch/develop/nightly/en/api_classifier.html